tests/common/test_logger.py

```diff
@@ -566,9 +566,9 @@ def test_file_appender(self):
             logcontent = logfile.readlines()
             # Levels are honored and Verbose should not be written.
             self.assertEqual(3, len(logcontent))
-            self.assertRegex(logcontent[0], r"(.*INFO\s*test-info.*)")
-            self.assertRegex(logcontent[1], r"(.*WARNING\s*test-warn.*)")
-            self.assertRegex(logcontent[2], r"(.*ERROR\s*test-error.*)")
+            self.assertRegex(logcontent[0], r"(.*INFO\s\w+\s*test-info.*)")
+            self.assertRegex(logcontent[1], r"(.*WARNING\s\w+\s*test-warn.*)")
```

Comment:
If possible, can you add a test with a dummy thread name added and verified that thread name comes in?